site/_js/web-components/tag-pill-list.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Given a key-value array, renders
+ * a list of tag-pills
+ */
+import {BaseElement} from './base-element';
+import {html} from 'lit-element';
+
+export class TagPillList extends BaseElement {
+  constructor() {
+    super();
+
+    this.items = [];
+    this.icon = this.querySelector('tag-pill-list-icon svg');
+  }
+
+  static get properties() {
+    return {
+      items: {type: Array, reflect: true},
+    };
+  }
+
+  render() {
+    const items = this.items.map(
+      item => html`
+        <span
+          class="surface hairline rounded-lg tag-pill type--label display-inline-flex align-center"
+          data-key="${item.key}"
+          data-value="${item.value}"
+        >
+          ${item.value} ${this.icon?.cloneNode(true)}
```

Comment:
I don't see any interactivity in this element. Why is it a web component and not just markup?
We aim to ship JS only if needed. It looks like here it is used to render some static HTML - we surely don't need Lit nor web components for that?

Reply:
So I'm using it for reactivity in this instance. If we don't use a WC then we'll need to render/append to the dom using JS anyway but happy to refactor to that if that's the preference

Reply:
I see, to make the list of pills dynamic? Can you make the example in the docs reactive then?
Also, I think the 'close' icon should do something.